spec/features/editing_content_settings/set_access_limit_spec.rb

```diff
@@ -58,11 +64,14 @@ def and_i_edit_the_access_limit
   end
 
   def and_i_limit_to_my_organisation
+    expect(page).to have_content("Primary org")
     choose I18n.t!("access_limit.edit.type.primary_organisation")
     click_on "Save"
   end
 
   def and_i_limit_to_tagged_organisations
+    expect(page).to have_content("Primary org", count: 2)
```

Comment:
Ah I see this is what you meant in your comment earlier - it does feel a bit of a loose test as any other places where we showed the organisation name would pass.

An alternative approach you could take is to check for "Primary org Supporting org" since that'll only match this grouping. 

Otherwise could use a within though that'd be a bit tricky to match the element, for example:

```
radio = page.find(:radio_button, I18n.t!("access_limit.edit.type.tagged_organisations"))
within radio.find(:xpath, "..") do 
 radio.choose
 expect(page).to have_content("Primary org")
 expect(page).to have_content("Supporting org")
end
```

Reply:
I agree. But I'd like to find another way:

   - The concatenated text approach did occur to me, but it just looks weird
   - The xpath approach also occurred, but it makes the tests more complex

I've pushed up a compromise that hopefully resolves the concern identified, although I can't say I like it much. This does seem like an integral part of the feature, so worth the testing effort 💪 .